static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap

```diff
@@ -1,4 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
+// update the snapshots with command: npm test -- -u
```

Comment:
can you move this to the README file https://github.com/datacommonsorg/website/blob/master/README.md

Here might be override when creating a new snapshot.

Reply:
Right. Done